app_dart/lib/src/foundation/utils.dart

```diff
@@ -68,6 +68,39 @@ Future<List<String>> loadBranchRegExps(
   return <String>['master'];
 }
 
+Future<List<String>> getBranchList(
+    Config config,
+    HttpClientProvider branchHttpClientProvider,
+    Logging log,
+    GitHubBackoffCalculator gitHubBackoffCalculator) async {
+  final List<String> regExps = await loadBranchRegExps(
+      branchHttpClientProvider, log, gitHubBackoffCalculator);
```

Comment:
Is this a temporary fix? we probably should avoid hardcoding branch names.

Reply:
If it is a temporary fix please add a todo with the bug number to remove it.

Reply:
Added that in this PR description. I will have a follow up PR soon.

Reply:
I think is better to add todos and bug number to the code. Is difficult to keep track of them from PR descriptions.

Reply:
Yeah. Just added: https://github.com/flutter/flutter/issues/54297